src/client/client.js

```diff
@@ -1,11 +1,13 @@
 import {render} from 'react-dom';
 import React from 'react';
-import {syncHistory, routeReducer} from 'redux-simple-router'
+import {syncHistory, routeReducer} from 'redux-simple-router';
 import {Map, fromJS} from 'immutable';
 import {ensureState} from 'redux-optimistic-ui';
 
-const makeStore = __PRODUCTION__ ? require('./makeStore.prod.js') : require('./makeStore.dev.js');
-const Root = __PRODUCTION__ ? require('./Root.prod.js') : require('./Root.dev.js');
+const ENV_SUFFIX = __PRODUCTION__ ? 'prod' : 'dev';
+
+const makeStore = require(`./makeStore.${ENV_SUFFIX}.js`);
```

Comment:
dynamic requires are no good. webpack must require everything that they could possibly be. That makes for a bigger production bundle.